analyzers/src/SonarAnalyzer.CSharp/Rules/Hotspots/DisablingCsrfProtection.cs

```diff
@@ -44,7 +44,7 @@ public sealed class DisablingCsrfProtection : HotspotDiagnosticAnalyzer
                         return;
                     }
 
-                    context.RegisterSyntaxNodeActionInNonGenerated(CheckIgnoreAntiforgeryTokenAttribute,
+                    context.RegisterNodeAction(CheckIgnoreAntiforgeryTokenAttribute,
                                                                    SyntaxKind.Attribute,
```

Comment:
the alignment is off

Reply:
Nice illustration of why I don't like these varying far indents :)